cinnamon-take-screenshot@duracell80/files/cinnamon-take-screenshot@duracell80/cinnamon-take-screenshot.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+# Take screenshots directly from the desktop
+# @git:duracell80
+
+
+# ENVIRONMENT VARS
+LANG="en"
+DIR_PWD=$(pwd)
+DIR_TGT="${HOME}/Pictures/screenshots"
```

Comment:
The path that the system/user has specified for saving screenshots using gnome-screenshot can be retrieved and used instead of hard-coded by using `gsettings get org.gnome.gnome-screenshot auto-save-directory`. Other applications also support similar user preferences.

Ultimately, it's your call for how to implement this but you should make clear in the README which approach you're using so that users can set their expectations appropriately.

Reply:
Hey thank you for the info, I didn't know that. Yeah that would give the end user more control over where the files are saved. In my current environment that is not set / is blank. Perhaps it could prefer the user setting but if blank then use this fallback folder in Pictures. 

Thanks for the help and for providing this great new spice possibility.

Reply:
Sorry for the oversight, if it is blank, that will default to `~/Pictures`. So if you want to use the check and then set it if the check returns null.

Thanks for the awesome contribution!

Reply:
No problem at all, thanks for helping make this more robust.

That should have been added to this pull now. I tested with ./test-spice also.